frontend/src/container/SideNav/NavItem/NavItem.tsx

```diff
@@ -5,6 +5,7 @@ import './NavItem.styles.scss';
 import { Tag } from 'antd';
 import cx from 'classnames';
 
+// import { Color } from '@signozhq/design-tokens';
```

Comment:
Remove commented code.

Reply:
I wanted to keep commented code on how to used design-tokens in future. Is that not recommended?

Reply:
removed commetns